vowpalwabbit/parse_example_json.h

```diff
@@ -1670,11 +1670,11 @@ inline void apply_pdrop(vw& all, float pdrop, v_array<example*>& examples)
 {
   if (all.example_parser->lbl_parser.label_type == label_type_t::cb)
```

Comment:
is this bug just relevant for json parsing?

Reply:
in the way that "pdrop" as concept exists only in json format.